src/Mapper/Builder/OptionsBuilder.php

```diff
@@ -63,155 +36,6 @@ public function add(string $key, $value): self
         return $this;
     }
 
-    public static function createOneToOne(string $fieldName, string $targetEntity): self
```

Comment:
Doing this on a stable branch is a BC break :(

Reply:
Yeah, my idea was to make it before having a release, but we made one yesterday, I'll change it